<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1333</width>
    <height>691</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Gestion des Clients</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <widget class="QPushButton" name="ajouter_Materiel">
    <property name="geometry">
     <rect>
      <x>350</x>
      <y>480</y>
      <width>171</width>
      <height>61</height>
     </rect>
    </property>
    <property name="text">
     <string>Ajouter</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="id_materiel">
    <property name="geometry">
     <rect>
      <x>170</x>
      <y>100</y>
      <width>113</width>
      <height>26</height>
     </rect>
    </property>
    <property name="text">
     <string>4546</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="nom_materiel">
    <property name="geometry">
     <rect>
      <x>170</x>
      <y>160</y>
      <width>113</width>
      <height>26</height>
     </rect>
    </property>
    <property name="text">
     <string>jbskj</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="etat_materiel">
    <property name="geometry">
     <rect>
      <x>170</x>
      <y>210</y>
      <width>113</width>
      <height>26</height>
     </rect>
    </property>
    <property name="text">
     <string>dns</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="date_achat_materiel">
    <property name="geometry">
     <rect>
      <x>170</x>
      <y>270</y>
      <width>113</width>
      <height>26</height>
     </rect>
    </property>
    <property name="text">
     <string>20/10</string>
    </property>
   </widget>
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>80</x>
      <y>100</y>
      <width>63</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>Idmatériel</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_2">
    <property name="geometry">
     <rect>
      <x>80</x>
      <y>160</y>
      <width>71</width>
      <height>41</height>
     </rect>
    </property>
    <property name="text">
     <string>Nom</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_3">
    <property name="geometry">
     <rect>
      <x>80</x>
      <y>210</y>
      <width>63</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>Etat</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_4">
    <property name="geometry">
     <rect>
      <x>70</x>
      <y>270</y>
      <width>63</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>Date d'achat</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="duree_garantie_materiel">
    <property name="geometry">
     <rect>
      <x>170</x>
      <y>330</y>
      <width>113</width>
      <height>26</height>
     </rect>
    </property>
    <property name="text">
     <string>2</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_5">
    <property name="geometry">
     <rect>
      <x>70</x>
      <y>340</y>
      <width>63</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>Duree Garantie</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="prix_materiel">
    <property name="geometry">
     <rect>
      <x>170</x>
      <y>390</y>
      <width>113</width>
      <height>26</height>
     </rect>
    </property>
    <property name="text">
     <string>2</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="duree_de_vie_materiel">
    <property name="geometry">
     <rect>
      <x>170</x>
      <y>510</y>
      <width>113</width>
      <height>26</height>
     </rect>
    </property>
    <property name="text">
     <string>2</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="quantite_materiel">
    <property name="geometry">
     <rect>
      <x>170</x>
      <y>450</y>
      <width>113</width>
      <height>26</height>
     </rect>
    </property>
    <property name="text">
     <string>2</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_6">
    <property name="geometry">
     <rect>
      <x>70</x>
      <y>390</y>
      <width>63</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>Prix</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_7">
    <property name="geometry">
     <rect>
      <x>70</x>
      <y>450</y>
      <width>63</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>Quantite dispo</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_8">
    <property name="geometry">
     <rect>
      <x>70</x>
      <y>510</y>
      <width>63</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>Duree de vie</string>
    </property>
   </widget>
   <widget class="QTableView" name="affichage">
    <property name="geometry">
     <rect>
      <x>330</x>
      <y>20</y>
      <width>871</width>
      <height>441</height>
     </rect>
    </property>
   </widget>
   <widget class="QLineEdit" name="id_supp">
    <property name="geometry">
     <rect>
      <x>1150</x>
      <y>490</y>
      <width>113</width>
      <height>26</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="label_9">
    <property name="geometry">
     <rect>
      <x>1050</x>
      <y>490</y>
      <width>63</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>Idmatériel</string>
    </property>
   </widget>
   <widget class="QPushButton" name="supression_Materiel">
    <property name="geometry">
     <rect>
      <x>550</x>
      <y>470</y>
      <width>171</width>
      <height>61</height>
     </rect>
    </property>
    <property name="text">
     <string>supression</string>
    </property>
   </widget>
   <widget class="QPushButton" name="modifier_Materiel">
    <property name="geometry">
     <rect>
      <x>750</x>
      <y>470</y>
      <width>171</width>
      <height>61</height>
     </rect>
    </property>
    <property name="text">
     <string>modifier</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="id_materiel_2">
    <property name="geometry">
     <rect>
      <x>1150</x>
      <y>520</y>
      <width>113</width>
      <height>26</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="label_10">
    <property name="geometry">
     <rect>
      <x>1060</x>
      <y>520</y>
      <width>63</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>id</string>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1333</width>
     <height>26</height>
    </rect>
   </property>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
